package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name":     "jshint",
-  "version":  "2.4.4",
```

Comment:
After re-reading README, the reason why forking from 2.4.4 has been explained there. Thanks.